Terminal.Gui/Views/ListView.cs

```diff
@@ -501,7 +527,12 @@ public virtual bool MoveEnd ()
 
             if (Viewport.Y + _selected > Viewport.Height - 1)
             {
-                Viewport = Viewport with { Y = _selected };
+                Viewport = Viewport with
```

Comment:
Aren't we using a different term than "Viewport" now?

I've lost track.

Reply:
Now we are using `Viewport`. Before was `Bounds`.